src/styles/blocks/_content.scss

```diff
@@ -0,0 +1,38 @@
+.body {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  align-items: center;
+}
+
+.main {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.container {
+  margin-top: 50px;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+
+  width: $card-block-width;
+
+}
+
+.card {
+  margin-right: 48px;
+  margin-bottom: 46px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+}
```

Comment:
1 - you've already used this class, it's strange that you set styles for it in two files
2 - you can't use margins for BEM-blocks (use mix of BEM block and element for this purpose)